tfe/data_source_policy_set.go

```diff
@@ -33,6 +33,19 @@ func dataSourceTFEPolicySet() *schema.Resource {
 				Computed: true,
 			},
 
+			"kind": {
+				Description: "The policy-as-code framework for the policy. Valid values are sentinel and opa",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+
+			"overridable": {
+				Description: "Whether users can override this policy when it fails during a run. Only valid for OPA policies",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
```

Comment:
I don't think a default is necessary in a data resource. Does this create certain behavior that you want?